src/main/js/helpers/legend.js

```diff
@@ -150,7 +150,8 @@ const createLegend = (container) =>
 const legendClickHandler = (element) => {
     const target = d3.select(element);
     const isSelected = target.attr("aria-current");
-    return target.attr("aria-current", isSelected !== "true");
+    target.attr("aria-current", isSelected !== "true");
+    return isSelected !== "true";
```

Comment:
Update documentation on Line 148 and maybe avoid doing `isSelected !== "true"` twice and instead make use of a variable.

Reply:
Updated here https://github.com/cerner/carbon-graphs/pull/140/commits/e6eaf858e8e58510cc2e5b8f189e32df80b4059d.